test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SignTestUtility.cs

```diff
@@ -14,21 +14,6 @@ namespace NuGet.Packaging.Test
 {
     public static class SignTestUtility
     {
-        // Environment variable for a valid RFC 3161 timestamping service.
-        private static readonly string _testTimestampServer = Environment.GetEnvironmentVariable("TIMESTAMP_SERVER_URL");
```

Comment:
Is there any value in integrating with a real TSA? We have a similar concept where we have integration tests for all sorts of known NuGet server implementations (e.g. NuGet.Server, running in Azure somewhere). Maybe just one happy path.

Reply:
I don't think so.  I think the test risk outweighs the benefit.  The real TSA could become unavailable, change its URL, change the certificate signature algorithm, etc.  I don't think these are good reasons to fail a test; we're not testing the TSA but that our timestamp code requests and processes RFC 3161 timestamps.  If there is an interesting scenario that a real TSA offers that isn't already covered here, I don't see why we can't capture it with this test server.

Eventually, the RFC 3161 timestamping classes in NuGet client will be replaced with classes from .NET Framework / .NET Core.